modules/nw-cfg-tuning-interface-cni.adoc

```diff
@@ -97,6 +97,7 @@ spec:
   - name: podexample
     image: centos
     command: ["/bin/bash", "-c", "sleep INF"]
+    # TODO: Do we need to update, or is it good?
```

Comment:
@s-urbaniak @stlaz There are a ton of files in the networking book files that already had some securityContext set and that I need help with if and how to update them (I'm not going to tag you on each of them - just search for "TODO"). 

Let me know if we need to pull in any other SMEs or QEs to answer these questions.